src/Command/ThanksCommand.php

```diff
@@ -39,10 +49,42 @@ class ThanksCommand extends BaseCommand
             'name' => 'laravel/laravel',
             'url' => 'https://github.com/laravel/laravel',
         ],
+        'nette' => [
+            'name' => 'nette/nette',
+            'url' => 'https://github.com/nette/nette',
+        ],
+        'phpDocumentor' => [
+            'name' => 'phpDocumentor/phpDocumentor2',
+            'url' => 'https://github.com/phpDocumentor/phpDocumentor2',
+        ],
+        'piwik' => [
+            'name' => 'piwik/piwik',
+            'url' => 'https://github.com/piwik/piwik',
+        ],
+        'reactphp' => [
+            'name' => 'reactphp/react',
+            'url' => 'https://github.com/reactphp/react',
+        ],
+        'sebastianbergmann' => [
+            'name' => 'phpunit/phpunit',
+            'url' => 'https://github.com/sebastianbergmann/phpunit',
+        ],
+        'slimphp' => [
+            'name' => 'slimphp/Slim',
+            'url' => 'https://github.com/slimphp/Slim',
+        ],
+        'Sylius' => [
+            'name' => 'Sylius/Sylius',
+            'url' => 'https://github.com/Sylius/Sylius',
+        ],
         'symfony' => [
```

Comment:
should we rather handle it in the `extra` key of Symfony subtree packages ? We have separate Symfony packages, which are separate from `symfony/symfony`

Reply:
that's a "second scale" detail, `symfony/*` (whether from subtree splits or not) are not wrong forwarding to `symfony/symfony` (same reasoning for other entries in this list)

Reply:
well, `symfony/polyfill-*` should rather forward to `symfony/polyfill` instead, as this is where this code comes from.